style.json

```diff
@@ -2060,13 +2060,14 @@
       "source": "openmaptiles",
       "source-layer": "place",
       "filter": [
-        "!in",
-        "class",
-        "city",
-        "town",
-        "village",
-        "country",
-        "continent"
+          "!in",
+          "class",
+          "city",
+          "town",
+          "village",
+          "state",
+          "country",
+          "continent"
```

Comment:
The new indent of this block is suspicious.

Reply:
fixed, thanks for pointing it out @frodrigo